_opensearch/query-dsl/full-text.md

```diff
@@ -5,14 +5,22 @@ parent: Query DSL
 nav_order: 40
 ---
 
-# Full-text queries
+# Full-text query types and options
 
-This page lists all full-text query types and common options. Given the sheer number of options and subtle behaviors, the best method of ensuring useful search results is to test different queries against representative indices and verify the output.
+This page lists all full-text query types and common options. There are many optional fields you can use to create subtle search behaviors, so we recommend that you test out some basic query types against representative indexes and verify the output before you do more advanced or complex searches with multiple options.
 
----
+OpenSearch uses the Apache Lucene search library that provides highly efficient data structures and algorithms for ingesting, indexing, searching, and aggregating data.
+
+To learn more about search query classes, see [Lucene query JavaDocs](https://lucene.apache.org/core/8_9_0/core/org/apache/lucene/search/Query.html).
 
-## Table of contents
+To convert unstructured text into the format that you need, you can use the text analyzers. To learn more about how to convert unstructured text into structured text that is optimized for search, see [Text analyzers]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/text-analyzers).
```

Comment:
This section can be omitted. The queries below analyze the search text automatically.

Reply:
sure. I do think this is helpful to point them to the page with more information about text analyzers. ok to leave it in? thanks

Reply:
Sure. But I think we need to differentiate between analyzing the fields when they are indexed and analyzing the search string in the query at query time. If we're referring to the first one, then it's ok to leave as-is but maybe add "at index time". If we're referring to the second one, we have to reword so that the users know that the analysis is happening automatically with all these queries. All they have to do is (optionally) provide an analyzer.

Reply:
thanks for the additional information. 
ok, i've updated it as follows to clarify: 
The full-text query types shown in this section use the standard analyzer that analyzes text automatically when the query is submitted.

You can also analyze fields when you index them. To learn more about how to convert unstructured text into structured text that is optimized for search, see [Text analyzers]...